backend/src/last-wills/lastwills.controller.ts

```diff
@@ -0,0 +1,139 @@
+import {
+  Body,
+  ClassSerializerInterceptor,
+  Controller,
+  Delete,
+  Get,
+  HttpCode,
+  HttpStatus,
+  Param,
+  Post,
+  Put,
+  Req,
+  SerializeOptions,
+  UseGuards,
+  UseInterceptors,
+} from '@nestjs/common'
+import {
+  ApiBadRequestResponse,
+  ApiBearerAuth,
+  ApiCreatedResponse,
+  ApiNoContentResponse,
+  ApiNotFoundResponse,
+  ApiOkResponse,
+  ApiTags,
+  ApiUnauthorizedResponse,
+} from '@nestjs/swagger'
+import { LastWill, LastWillMetadata } from '../db/entities/lastwill.entity'
+import { LastWillsService } from '../db/services/lastwills.service'
+import { JwtGuard } from '../shared/guards/jwt.guard'
+import { RequestWithJWTPayload } from '../shared/interfaces/request-with-user.interface'
+import { CreateLastWillDto } from './dto/create-lastwill.dto'
+import { UpdateLastWillDto } from './dto/update-lastwill.dto'
+
+@UseGuards(JwtGuard)
+@ApiTags('lastwill')
+@Controller('lastwill')
+@UseInterceptors(ClassSerializerInterceptor)
+@SerializeOptions({ strategy: 'excludeAll' })
+@ApiBearerAuth('access_token')
+export class LastWillsController {
```

Comment:
Add `@ApiOperation` to the endpoints. Afaik the other controllers have it as well